airflow/jobs/local_task_job.py

```diff
@@ -293,5 +295,6 @@ def heartbeat_callback(self, session: Session = NEW_SESSION) -> None:
 
     def _log_return_code_metric(self, return_code: int):
         Stats.incr(
-            f"local_task_job.task_exit.{self.id}.{self.dag_id}.{self.task_instance.task_id}.{return_code}"
+            f"local_task_job.task_exit."
```

Comment:
```suggestion
            "local_task_job.task_exit."
```

Reply:
Strange it was not raised by ruff :(. I think flake was doing it before.